openrtb_ext/imp_conversant.go

```diff
@@ -0,0 +1,14 @@
+package openrtb_ext
+
+type ExtImpConversant struct {
+	SiteID      string   `json:"site_id"`
+	Secure      *int8    `json:"secure"`
+	TagID       string   `json:"tag_id"`
+	Position    *int8    `json:"position"`
+	BidFloor    float64  `json:"bidfloor"`
+	Mobile      *int8    `json:"mobile"`
```

Comment:
Mobile seemed to be unused?

Reply:
I can remove that from the adapter and this file.  anywhere else it should be removed from